src/components/tooltip/tooltip.tsx

```diff
@@ -71,6 +72,13 @@ export class Tooltip {
     @Prop({ reflect: true })
     public elementId!: string;
 
+    /**
+     * The maximum amount of characters before rendering 'label' and 'helperLabel' in two rows.
+     * Defaults to 50 characters.
+     */
+    @Prop({ reflect: true })
+    public maxCharacterLength: number;
```

Comment:
Documenting the default like this can be prone to errors if we decide to change the value in the future. It can be very easy to forget to either update the documentation or the code itself. 

If we instead give the prop a default value, we can both remove the documentation about the default (since it will be documented automatically) and remove the code below in the render method.

```suggestion
     */
    @Prop({ reflect: true })
    public maxCharacterLength: number = DEFAULT_MAX_LENGTH;
```

(`DEFAULT_MAX_LENGTH` can be defined outside the component so we don't need to turn of the eslint rule about magic numbers)

Reply:
We might also want to just name this `maxlength` to keep it aligned with the naming of `limel-input-field`

Reply:
Great feedback! I t actually happened to us too to forget updating it, when we changed the default.

Will fix :)

Reply:
Fixed ;)